features/git-sync/all_branches/feature_branch_merge_main_branch_conflict/without_remote_origin/first_branch/with_open_changes.feature

```diff
@@ -0,0 +1,122 @@
+      | main     | local    | main commit     | conflicting_file |
+      | feature1 | local    | feature1 commit | conflicting_file |
+      | feature2 | local    | feature2 commit | feature2_file    |
+
+
+  Scenario: skipping
+    When I run `git sync --skip`
+    Then it runs the Git commands
+      | BRANCH   | COMMAND                  |
+      | feature1 | git merge --abort        |
+      | feature1 | git checkout feature2    |
+      | feature2 | git merge --no-edit main |
+      | feature2 | git checkout main        |
+      | main     | git stash pop            |
+    And I end up on the "main" branch
+    And I again have an uncommitted file with name: "uncommitted" and content: "stuff"
+    And I have the following commits
+      | BRANCH   | LOCATION | MESSAGE                           | FILE NAME        |
+      | main     | local    | main commit                       | conflicting_file |
+      | feature1 | local    | feature1 commit                   | conflicting_file |
+      | feature2 | local    | feature2 commit                   | feature2_file    |
+      |          |          | main commit                       | conflicting_file |
+      |          |          | Merge branch 'main' into feature2 |                  |
+
+
+  @finishes-with-non-empty-stash
+  Scenario: continuing without resolving the conflicts
+    When I run `git sync --continue`
+    Then it runs no Git commands
+    And I get the error "You must resolve the conflicts before continuing the git sync"
+    And I am still on the "feature1" branch
+    And I don't have an uncommitted file with name: "uncommitted"
+    And my repo still has a merge in progress
+
+
+  Scenario: continuing after resolving the conflicts
+    Given I resolve the conflict in "conflicting_file"
+    And I run `git sync --continue`
+    Then it runs the Git commands
+      | BRANCH   | COMMAND                  |
+      | feature1 | git commit --no-edit     |
+      | feature1 | git checkout feature2    |
+      | feature2 | git merge --no-edit main |
+      | feature2 | git checkout main        |
+      | main     | git stash pop            |
+    And I end up on the "main" branch
+    And I again have an uncommitted file with name: "uncommitted" and content: "stuff"
+    And I have the following commits
+      | BRANCH   | LOCATION | MESSAGE                           | FILE NAME        |
+      | main     | local    | main commit                       | conflicting_file |
+      | feature1 | local    | feature1 commit                   | conflicting_file |
+      |          |          | main commit                       | conflicting_file |
+      |          |          | Merge branch 'main' into feature1 |                  |
+      | feature2 | local    | feature2 commit                   | feature2_file    |
+      |          |          | main commit                       | conflicting_file |
+      |          |          | Merge branch 'main' into feature2 |                  |
+
```

Comment:
Please also specify the expected content of all files here. This is something that I am currently working on to add to all other specs (an example is [here](https://github.com/Originate/git-town/blob/kg-file-content-verifications/features/git-sync/all_branches/feature_branch_merge_main_branch_conflict/with_tracking_branch_updates/first_branch/with_open_changes.feature#L53-L58), and it would be nice to have it here as well. 

```cucumber
And now I have the following committed files
  | BRANCH   | NAME             | CONTENT          |
  | main     | conflicting_file | main content     |
  | feature1 | conflicting_file | resolved content |
  | feature2 | feature2_file    | feature2 content |
```